itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/TestMetastoreVersionInfo.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hive.metastore;
```

Comment:
The `MetastoreVersionInfo` class is in the `standalone-metastore` module but this test is in `hive-unit`. This is does not seem to be an integration test and it seems that it does not require anything more than the `MetastoreVersionInfo` class. Normally unit tests should be in the same module and package with the classes they are testing.

Reply:
Agree, moved to org.apache.hadoop.hive.metastore.utils